src/main/windows/callsWidgetWindow.ts

```diff
@@ -121,8 +121,8 @@ export default class CallsWidgetWindow extends EventEmitter {
     }
 
     private getWidgetURL() {
-        const u = new url.URL(this.config.siteURL);
-        u.pathname += `/plugins/${CALLS_PLUGIN_ID}/standalone/widget.html`;
+        const u = new url.URL(this.mainView.serverInfo.server.url.toString());
+        u.pathname += `plugins/${CALLS_PLUGIN_ID}/standalone/widget.html`;
```

Comment:
For server URLs with a pathname (eg. `https://subpath.test.mattermost.com/mattermost`) there will likely not be a trailing slash, so the widget URL here won't work. We have a function in `common/utils/url` called `getFormattedPathName` that should help make this more predictable.

Might also be worth adding a unit test for this case as well.